### ISSUE TITLE
How to run a mxnet example using GPU on Jetson TX1?

### ISSUE THREAD

**Issue by wangdi1983** [apache/incubator-mxnet#9134]
1. When I try running a mxnet example using GPU on my Jetson TX1 board, such as:
         nvidia@tegra-ubuntu:~/mxnet/example/image-classification$ python train_imagenet.py --gpus 0 
    It failed with some mistake, as an attaching file.

2. ThenI try running it without using GPU:
         nvidia@tegra-ubuntu:~/mxnet/example/image-classification$ python train_imagenet.py
   It successed.  But When I trying to watch GPU info:
         nvidia@tegra-ubuntu:~/mxnet/example/image-classification$ sudo ~/tegrastats 
   I got:
         RAM 1214/3983MB (lfb 19x4MB) cpu [13%,61%,52%,30%]@1734 EMC 5%@1600 APE 25 GR3D 0%@998
RAM 1214/3983MB (lfb 19x4MB) cpu [12%,45%,62%,37%]@1734 EMC 5%@1600 APE 25 GR3D 0%@998
RAM 1214/3983MB (lfb 19x4MB) cpu [30%,44%,61%,23%]@1734 EMC 5%@1600 APE 25 GR3D 0%@998
RAM 1214/3983MB (lfb 19x4MB) cpu [42%,44%,45%,19%]@1734 EMC 5%@1600 APE 25 GR3D 0%@998
RAM 1214/3983MB (lfb 19x4MB) cpu [26%,40%,59%,30%]@1734 EMC 5%@1600 APE 25 GR3D 0%@998
################################################################################

Dos "GR3D 0%@998" means GPU not working?  And how to run a mxnet example using GPU on Jetson TX1?

Hoping for an answer...

**Comment by KellenSunderland** [comment]
Hey @wangdi1983 thanks for bringing this to our attention.  I'm not sure why this file would be in a different folder on the TX1 as opposed to a GPU instance.  We can look into it but it may take a few days as many community members are currently visiting family and not working actively on the project.

"Dos "GR3D 0%@998" means GPU not working? And how to run a mxnet example using GPU on Jetson TX1?"
Yes, I believe this means that your GPU is clocked at 998 Mhz and is currently 0% active.

**Comment by wangdi1983** [comment]
@KellenSunderland  Thanks for your attention and hoping for your answer

**Comment by sandeep-krishnamurthy** [comment]
@KellenSunderland - Any updates on the issue? Thanks.

**Comment by ThomasDelteil** [comment]
@KellenSunderland, @larroy, @marcoabreu  do you have a JetsonTX1 available to run the above example and confirm it is failing?

**Comment by larroy** [comment]
Of course. We will add it to our queue and plan to work on this. @wangdi1983  how did you compile / get MXNet?
